.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+
+## Use case
+
+<!-- What is the final use case? Describe it with words, code or even sandboxes (4ex: [codesandbox.io](https://codesandbox.io)) -->
+
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+- [x] _Bug fix_ (non-breaking change which fixes an issue)
+- [x] _New feature_ (non-breaking change which adds functionality)
+- [x] **_Breaking change_** (fix or feature that would cause existing functionality to not work as before)
+
+# Checklist
+
+<!-- Check all the items your PR meets. In general - only coverage item is allowed not to be checked. -->
+
+- [ ] Read the [Contributing Guide](https://github.com/streamich/react-use/blob/master/CONTRIBUTING.md)
+- [ ] Perform a code self-review
+- [ ] Comment the code, particularly in hard-to-understand areas
+- [ ] Update/add the documentation
+- [ ] Create the story
+- [ ] Write tests
+- [ ] Ensure the test suite passes (`yarn test`)
+- [ ] Provide 100% tests coverage
+- [ ] Make sure code lints (`yarn lint`). Fix it with `yarn lint:fix` in case of problems.
+- [ ] Make sure types are fine (`yarn lint:types`).
+
+<!-- If you can't check all the checkboxes - create a Draft PR and get back to it when you will be able to put some marks in list. -->
```

Comment:
Do we need this? If contributor does not check all checkboxes, we can see that from the checkbox list.

```suggestion

```

Reply:
The main idea is to notify author that it is better to create Draft PR and update it, so others will know about that pending work.